components/browser/engine-system/build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     implementation project(':support-utils')
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:${rootProject.ext.dependencies['kotlin']}"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:${rootProject.ext.dependencies['coroutines']}"
```

Comment:
I need to update this before #636 lands (or the other way around).

Reply:
Ah OK, I will do it. I will merge #636 and update this.